webapp/src/pages/offer/components/OfferComponent.js

```diff
@@ -2,10 +2,12 @@ import React, {Component} from "react";
 import {withRouter} from "react-router";
 import TextArea from "react";
 import {Chart} from "react-google-charts";
+import { userService } from "../../../services/user";
 import { offerServices  } from "../../../services/offer/offer.service";
 import { profileService} from "../../../services/profilelist/profilelist.service";
 
 const DEFAULT_EVENT_ID = process.env.REACT_APP_DEFAULT_EVENT_ID || 1;
+let DEFAULT_USER_ID = process.env.REACT_APP_DEFAULT_USER_ID || 1;
```

Comment:
There is no such thing as a default user ID? 
Also REACT_APP_DEFAULT_USER_ID.
Please remove this.
What you're saying, is that if a user isn't return, we randomly default to user 1? That makes no sense

The app has a default EVENT_ID because the "Deep Learning Indaba" and it has ID 1

Reply:
@LindanyRetroRabbit Reminder to remove this